build/Build.CI.GitHubActions.cs

```diff
@@ -12,18 +12,18 @@
 [CustomGitHubActions("pr_validation",
     GitHubActionsImage.WindowsLatest,
     GitHubActionsImage.UbuntuLatest,
-    AutoGenerate = false,
+    AutoGenerate = true,
     OnPushBranches = new[] { "master", "dev" },
     OnPullRequestBranches = new[] { "master", "dev" },
-    InvokedTargets = new[] { nameof(All) },
+    InvokedTargets = new[] { nameof(RunTests), nameof(CreateNuGet) },
     PublishArtifacts = true,
     EnableGitHubToken = true)
 ]
 
 [CustomGitHubActions("Windows_release",
     GitHubActionsImage.WindowsLatest,
     OnPushTags = new[] { "*" },
-    AutoGenerate = false,
+    AutoGenerate = true,
```

Comment:
Set this to false - we turned it off because it constantly regenerated these files. Repeat for all of the other templates @eaba